sycl/plugins/level_zero/pi_level_zero.cpp

```diff
@@ -3505,8 +3505,10 @@ pi_result piKernelCreate(pi_program Program, const char *KernelName,
     return PI_ERROR_UNKNOWN;
   }
 
-  // Update the refcount of the program to show its use by this kernel.
-  PI_CALL(piProgramRetain(Program));
+  // Update the refcount of the program and context to show it's used by this
+  // kernel.
+  { PI_CALL(piProgramRetain(Program)); }
```

Comment:
theses extra {} shouldn't be necessary as I've recently added ones to PI_CALL itself

Reply:
Thx for fixing this, removed braces.